server/lib/fusor/multilog.rb

```diff
@@ -57,20 +89,40 @@ def collect(base_path)
   end
 
   def stop_collect(base_path)
-    if check_nested_key(SETTINGS, [:fusor, :system, :logging, :watch])
-      SETTINGS[:fusor][:system][:logging][:watch].each do |entry|
-        file = File.join(base_path, entry[:file])
-
-        procs = `ps -elf | grep "#{file}" | grep -v "grep"`
-        procs.split("\n").each do |p|
-          pid = p.split(" ")[3].to_i
-          terminate(pid)
+    if "development".eql? Rails.env
```

Comment:
Not sure if this would work, but this looks pretty similar to the above.  What do you think about passing in the settings to a function and share the code? 

```ruby
def stop_collect(base_path)
  if "development".eql? Rails.env
    if check_nested_key(SETTINGS, [:fusor, :system, :logging, :watch, :development])
      terminate_collect_procs SETTINGS[:fusor][:system][:logging][:watch][:development]
    end
  elsif "production".eql? Rails.env
    if check_nested_key(SETTINGS, [:fusor, :system, :logging, :watch, :production])
      terminate_collect_procs SETTINGS[:fusor][:system][:logging][:watch][:production]
    end
  end
end
```  
```ruby
def terminate_collect_procs(settings)
  settings.each do |entry|
    file = File.join(base_path, entry[:file])

    procs = `ps -elf | grep "#{file}" | grep -v "grep"`
    procs.split("\n").each do |p|
      pid = p.split(" ")[3].to_i
      terminate(pid)
    end

    procs = `ps -elf | grep "#{entry[:file]}" | grep -v "grep"`
    procs.split("\n").each do |p|
      pid = p.split(" ")[3].to_i
      terminate(pid)
    end
  end
end
```